qc/workflow_exome.cwl

```diff
@@ -75,10 +75,40 @@ steps:
             metric_accumulation_level: picard_metric_accumulation_level
             bait_intervals: bait_intervals
             target_intervals: target_intervals
-            per_target_coverage: collect_hs_metrics_per_target_coverage
-            per_base_coverage: collect_hs_metrics_per_base_coverage
+            per_target_coverage:
+                default: false
+            per_base_coverage:
+                default: false
+        out:
+            [hs_metrics]
+    collect_per_base_metrics:
+        run: collect_hs_metrics.cwl
+        in:
+            cram: cram
+            reference: reference
+            metric_accumulation_level: picard_metric_accumulation_level
+            bait_intervals: bait_intervals
+            target_intervals: per_base_intervals
+            per_target_coverage:
+                default: false
+            per_base_coverage:
+                default: true
+        out:
+            [per_base_coverage_metrics]
+    collect_per_target_metrics:
+        run: collect_hs_metrics.cwl
+        in:
+            cram: cram
+            reference: reference
+            metric_accumulation_level: picard_metric_accumulation_level
+            bait_intervals: bait_intervals
```

Comment:
same comment as above, but use the per_target_intervals as bait_intervals input.